TODO.md

```diff
@@ -1,20 +1,12 @@
 # TODO
 
-## 2014-03-13
-
-- ending phase 1
-
-  - combined messages, like plan
-    - @neo unlock
-    - @neo apply
-  - @ellison fix test suite -- update to new tag style, used in combined plan
-    summary
```

Comment:
Why are these being removed?

Reply:
The combined messages is done. 
I believe @ellisonmarks fixed the test suite.